docs/excel/excel-add-ins-charts.md

```diff
@@ -15,48 +15,50 @@ For the complete list of properties and methods that the `Chart` and `ChartColle
 The following code sample creates a chart in the worksheet named **Sample**. The chart is a **Line** chart that is based upon data in the range **A1:B13**.
 
 ```js
-Excel.run(function (context) {
-    var sheet = context.workbook.worksheets.getItem("Sample");
-    var dataRange = sheet.getRange("A1:B13");
-    var chart = sheet.charts.add("Line", dataRange, "auto");
+await Excel.run(async (context) => {
+    let sheet = context.workbook.worksheets.getItem("Sample");
```

Comment:
I opted to do `let` because it'd work universally without risking introducing new errors. While I did (at minimum) test every sample for compilation, I didn't want to miss a scenario where two samples are assumed to be combined and the `const` would cause problems.